package.json

```
@@ -32,7 +44,7 @@
     "workbox-streams": "^5.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "nodemon ./src/server/app.js",
```

클라이언트의 start를 지우시면 안돼요... 서버용이라면 따로 하나 만들어주세요.

---

```suggestion
    "server": "nodemon ./src/server/app.js",
```

이렇게 새로 만들고

```
npm run server
```

이렇게 실행하면됩니다.